library/std/src/sys/windows/c.rs

```diff
@@ -1070,6 +1074,14 @@ extern "system" {
     pub fn ReleaseSRWLockShared(SRWLock: PSRWLOCK);
     pub fn TryAcquireSRWLockExclusive(SRWLock: PSRWLOCK) -> BOOLEAN;
     pub fn TryAcquireSRWLockShared(SRWLock: PSRWLOCK) -> BOOLEAN;
+
+    pub fn CompareStringOrdinal(
+        lpString1: LPCWSTR,
+        cchCount1: c_int,
+        lpString2: LPCWSTR,
+        cchCount2: c_int,
+        bIgnoreCase: i32,
```

Comment:
```suggestion
        bIgnoreCase: BOOL,
```

Small nit: I would use the type alias `BOOL` to match the signature in the [docs](https://docs.microsoft.com/en-us/windows/win32/api/stringapiset/nf-stringapiset-comparestringordinal) and make it clear that `c::TRUE` and `c::FALSE` are the only allowed options.